GitHub Actions broken due to "No module named 'sklearn'"

Probably, `sklearn` is missed from the GH Actions configuration.

```
 E   ModuleNotFoundError: No module named 'sklearn'
=========================== short test summary info ============================
ERROR tests/test_coniferest.py
ERROR tests/test_isoforest.py
!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!
============================== 2 errors in 0.43s ===============================
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -6,3 +6,34 @@ requires = [
   "Cython",
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "coniferest"
+version = "0.0.4"
+description = "Coniferous forests for better machine learning"
+readme = "README.md"
+requires-python = ">=3.7"
+authors = [
+  { name = "Vladimir Korolev", email = "balodja@gmail.com" },
+  { name = "SNAD team" },
+]
+license = { text = "MIT" }
+classifiers =  [
+  "License :: OSI Approved :: MIT License",
+  "Intended Audience :: Science/Research",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: MacOS",
+  "Programming Language :: Python",
+  "Topic :: Scientific/Engineering",
+]
+
+[project.urls]
+"Source Code" = "https://github.com/snad-space/coniferest"
+
+[tool.setuptools.packages.find]
+exclude = [
+  "tests",
+  "notebooks",
+  "notes",
+]

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -27,6 +27,11 @@ classifiers =  [
   "Programming Language :: Python",
   "Topic :: Scientific/Engineering",
 ]
+dependencies = [
+  "numpy",
+  "scikit-learn",
+  "matplotlib",
+]
 
 [project.urls]
 "Source Code" = "https://github.com/snad-space/coniferest"

--- a/setup.py
+++ b/setup.py
@@ -23,38 +23,10 @@ extensions = [Extension("coniferest.calc_paths_sum",
                         )]
 
 
-def get_readme():
-    return (Path(__file__).parent / 'README.md').read_text(encoding='utf8')
-
 
 setup(
-      version='0.0.4',
-      description='Coniferous forests for better machine learning',
-      long_description=get_readme(),
-      long_description_content_type='text/markdown',
-      url='https://github.com/snad-space/coniferest',
-      author='Vladimir Korolev, SNAD team',
-      author_email='balodja@gmail.com',
-      license='MIT',
-      classifiers=[
-        'Development Status :: 3 - Alpha',
-        'License :: OSI Approved :: MIT License',
-        'Intended Audience :: Science/Research',
-        'Natural Language :: English',
-        'Operating System :: Microsoft :: Windows',
-        'Operating System :: POSIX :: Linux',
-        'Operating System :: MacOS',
-        'Programming Language :: Python',
-        'Topic :: Scientific/Engineering'
-      ],
-      packages=['coniferest', 'coniferest.sklearn'],
-      package_data={
-          '': ['*.pxd'],
-      },
       ext_modules=cythonize(extensions),
-      install_requires=['numpy', 'sklearn', 'matplotlib'],
       cmdclass = {
           'build_ext': build_ext
       },
-      zip_safe=False)
-
+)

